app/controllers/vaccinations_controller.rb

```diff
@@ -7,7 +7,7 @@ class VaccinationsController < ApplicationController
   before_action :set_consent_response, only: %i[show confirm]
   before_action :set_triage, only: %i[show confirm]
 
-  layout "two_thirds"
+  layout "full_width"
```

Comment:
Isn't this equivalent to no special layout?

```suggestion
```

Reply:
I thought we needed this to add the flash ... ?

Reply:
> I thought we needed this to add the flash ... ?

we shouldn't need to.

i also think we should leave this at two-thirds

Reply:
Back to two-thirds it is!